src/styles/globals.css

```
@@ -0,0 +1,32 @@
+  outline: 2px solid var(--zp-color-focus-ring);
+  outline-offset: 2px;
+}
+
+.zp-visually-hidden {
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  padding: 0;
+  margin: -1px;
+  overflow: hidden;
+  clip: rect(0, 0, 0, 0);
```

Stylelint (17.7.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)